data_wrangling/inconsistent.ipynb

```diff
@@ -0,0 +1,466 @@
+{
```

Comment:
<p>There should be more comments here to explain what you're trying to do</p><p> </p><p>Same applies with the next 3 columns</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Data-Epic/gbotemi_bolarinwa/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>